torch/quantization/_quantize_script.py

```diff
@@ -62,6 +62,16 @@ def __setstate__(self, state):
     linear_packed_params = torch.jit.script(torch.nn.quantized.modules.linear.LinearPackedParams())._c
     conv_packed_params = torch.jit.script(ConvPackedParams())._c
 
+def _preprocess_qconfig_dict_helper(module, module_types, qconfig_dict, prefix=''):
+    dict_w_names = {}
+    for name, child in module.named_children():
+        module_prefix = prefix + '.' + name if prefix else name
+        if type(child) in module_types:
+            dict_w_names[module_prefix] = qconfig_dict[type(child)]
+        else:
+            dict_w_names.update(_preprocess_qconfig_dict_helper(child, module_types, qconfig_dict, module_prefix))
```

Comment:
nit: dict_w_names -> dict_with_names
also could you pass this as an argument and update the dictionary directly?

Reply:
actually could you just use qconfig_dict?